api/server/sdk/volume_node_ops.go

```diff
@@ -96,20 +91,17 @@ func (s *VolumeServer) Detach(
 		return nil, err
 	}
 
-	// Check if already attached
-	_, err := util.VolumeFromName(s.driver(), req.GetVolumeId())
```

Comment:
This code needed to be removed in a previous PR, but it was not. So I'm cleaning it up now, since it does nothing.